base_partner_sequence/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2004-2009 Tiny SPRL (<http://tiny.be>).
+# Copyright 2013 initOS GmbH & Co. KG (<http://www.initos.com>).
+# Copyright 2016 Tecnativa - Vicent Cubells
+# Copyright 2016 Camptocamp - Akim Juillerat (<http://www.camptocamp.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "author": "Tiny/initOS GmbH & Co. KG,"
+              "ACSONE SA/NV,"
+              "Tecnativa, "
+              "Odoo Community Association (OCA), "
+              "Camptocamp, ",
+    "name": "Add a sequence on customers' code",
+    "version": "12.0.1.0.0",
+    "category": "Generic Modules/Base",
+    "website": "http://www.initos.com",
```

Comment:
@CasVissers `website` URL replace as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)